code/datums/components/riding/riding_vehicle.dm

```diff
@@ -177,8 +179,81 @@
 	set_vehicle_dir_layer(EAST, OBJ_LAYER)
 	set_vehicle_dir_layer(WEST, OBJ_LAYER)
 
+/datum/component/riding/vehicle/scooter/skateboard/RegisterWithParent()
+	. = ..()
+	RegisterSignal(parent, COMSIG_ATOM_EXAMINE, PROC_REF(on_examine))
```

Comment:
Seems like the only thing we add on examine is information about slowing down, but being able to slow down is a boolean.
Is it not going to print that in situations where it is not true?